data/data/libvirt/main.tf

```diff
@@ -108,7 +109,7 @@ resource "libvirt_domain" "master" {
 
   network_interface {
     network_id = libvirt_network.net.id
-    hostname   = "${var.cluster_id}-master-${count.index}"
+    hostname   = "${var.cluster_id}-master-${count.index}.${var.cluster_domain}"
```

Comment:
does this affect the adoption by the machine-api?

Reply:
or how the node names are created?

Reply:
No, it doesn't but maybe we should do the same there for consistency.

Reply:
In any case, the provider change shouldn't be a blocker here.

Reply:
https://github.com/openshift/cluster-api-provider-libvirt/issues/166